src/adUnit/VastAdUnit.js

```diff
@@ -202,6 +208,34 @@ class VastAdUnit extends VideoAdUnit {
     this[_protected].finish();
   }
 
+  /**
+   * Returns the duration of the ad Creative or 0 if there is no creative.
+   *
+   * @returns {number} - the duration of the ad unit.
+   */
+  duration () {
+    if (!this.isStarted()) {
+      return 0;
+    }
+    const {videoElement} = this.videoAdContainer;
+
+    return videoElement.duration;
```

Comment:
no need for intermediate constant. Just `return this.videoAdContainer.videoElement.duration` is sufficient?

Reply:
👍